mesop/server/server.py

```diff
@@ -270,3 +281,9 @@ def handle_message(message):
     flask_app.socketio = socketio  # type: ignore
 
   return flask_app
+
+
+def get_static_folder() -> str | None:
+  if not app_config.static_folder:
+    return None
+  return safe_join(os.getcwd(), app_config.static_folder)
```

Comment:
Wasn't sure how best to support runfiles for this case. I feel like using os.getcwd() worked in bazel dev by chance.

Reply:
Yeah, it was weird. I tried that initially, but it kept erroring out due to the base path being different than I was expecting.

```
FileNotFoundError: [Errno 2] No such file or directory: '/private/var/tmp/_bazel_/4fbe097e24ba789ddebddac4ca5258ab/execroot/mesop/bazel-out/darwin_arm64-fastbuild/bin/mesop/web/src/app/editor/web_package/static'
```

I was expecting it to be:

```
'/private/var/tmp/_bazel_/4fbe097e24ba789ddebddac4ca5258ab/execroot/mesop/bazel-out/darwin_arm64-fastbuild/bin/mesop/mesop/static'
```

But I'll try to test if os.cwd works downstream and for pip install version. I'm not too confident about it.

Reply:
One thing that makes me a little nervous is that we're using `os.getcwd()` - although we're already using something similar for web components serving, in that case we have a file extension check.

There could be a security issue if a Mesop app developer (for some reason) has the cwd be in some top-level directory (e.g. `/`) and they do something like this:

```
MESOP_STATIC_FOLDER="usr" mesop /usr/home/local/app.py
```

This is a somewhat contrived example, but I think we should clearly document this both in the guide and api docs. 

I think resolving this relative to the Mesop app entrypoint file could be even better (both here and for web component serving), but that's a larger change and I think something we could defer as long as we clearly document for now. 

For example, here's how flask resolves to the root path (which is what they use for resolving relative paths for static_folder):

https://github.com/pallets/flask/blob/dffe3034825edc4f84aded013fc9c6590e545ed5/src/flask/helpers.py#L569

It seems non-trivial and even then they are falling back to os.getcwd() in some cases.